code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1027,27 +1027,32 @@
 ///Returns a number between -1 to 2
 /mob/living/carbon/human/eyecheck()
 	var/number = 0
-	if(istype(src.head, /obj/item/clothing/head/welding))
-		if(!src.head:up)
-			number += 2
-	if(istype(src.head, /obj/item/clothing/head/helmet/space))
-		number += 2
-	if(istype(src.head, /obj/item/clothing/head/helmet/space/eva))
-		number -= 2
-	if(istype(src.head, /obj/item/clothing/head/helmet/space/rig/medical))
-		number -= 2
-	if(istype(src.glasses, /obj/item/clothing/glasses/thermal))
-		number -= 1
-	if(istype(src.glasses, /obj/item/clothing/glasses/sunglasses))
-		number += 1
-	if(istype(src.glasses, /obj/item/clothing/glasses/hud/health_advanced)) // New blueshield / brig phys no flash medi hud

```

Comment:
You'll need to set the propre level for EVA suits, the medical rig, and the advanced health HUD, too.

Might want to go through TG's code to see what else they have on what to make sure it matches up.